services/database/relational/tests/ccc-rdms-c1.feature

```diff
@@ -0,0 +1,69 @@
+@CCC.RDMS.C1
+Feature: Role-based access control and resource restriction
+
+"""
+This feature verifies that access controls are in place to enforce role-based access, restrict access to database resources based on role definitions, and confirm that unauthorized roles cannot access database resources.
+"""
+
+@CCC.RDMS.C1.TR01
```

Comment:
In the object storage catalog each of the testing requirements map to a gherkin feature (not a scenario). See here: https://github.com/mlysaght2017/common-cloud-controls/blob/main/services/storage/object/tests/ccc-os-c1.feature#L2

And each of the individual tests within the testing requirement feature map to gherkin scenarios 

Can we keep that alignment here too?

Reply:
Done. Please check.